src/pages/list/basic-list/index.tsx

```diff
@@ -0,0 +1,378 @@
+        </div>
+      </div>
+    );
+
+    const MoreBtn: React.FC<{
+      item: BasicListItemDataType;
+    }> = ({ item }) => (
+      <Dropdown
+        overlay={
+          <Menu onClick={({ key }) => editAndDelete(key, item)}>
+            <Menu.Item key="edit">编辑</Menu.Item>
+            <Menu.Item key="delete">删除</Menu.Item>
+          </Menu>
+        }
+      >
+        <a>
+          更多 <DownloadOutlined />
+        </a>
+      </Dropdown>
+    );
+
+    const getModalContent = () => {
+      if (done) {
+        return (
+          <Result
+            status="success"
+            title="操作成功"
+            subTitle="一系列的信息描述，很短同样也可以带标点。"
+            extra={
+              <Button type="primary" onClick={this.handleDone}>
+                知道了
+              </Button>
+            }
+            className={styles.formResult}
+          />
+        );
+      }
+      return (
+        <Form onSubmit={this.handleSubmit}>
+          <FormItem label="任务名称" {...this.formLayout}>
+            {getFieldDecorator('title', {
+              rules: [{ required: true, message: '请输入任务名称' }],
+              initialValue: current.title,
+            })(<Input placeholder="请输入" />)}
+          </FormItem>
+          <FormItem label="开始时间" {...this.formLayout}>
+            {getFieldDecorator('createdAt', {
+              rules: [{ required: true, message: '请选择开始时间' }],
+              initialValue: current.createdAt ? moment(current.createdAt) : null,
+            })(
+              <DatePicker
+                showTime
+                placeholder="请选择"
+                format="YYYY-MM-DD HH:mm:ss"
+                style={{ width: '100%' }}
+              />,
+            )}
+          </FormItem>
+          <FormItem label="任务负责人" {...this.formLayout}>
+            {getFieldDecorator('owner', {
+              rules: [{ required: true, message: '请选择任务负责人' }],
+              initialValue: current.owner,
+            })(
+              <Select placeholder="请选择">
+                <SelectOption value="付晓晓">付晓晓</SelectOption>
+                <SelectOption value="周毛毛">周毛毛</SelectOption>
+              </Select>,
+            )}
+          </FormItem>
+          <FormItem {...this.formLayout} label="产品描述">
+            {getFieldDecorator('subDescription', {
+              rules: [{ message: '请输入至少五个字符的产品描述！', min: 5 }],
+              initialValue: current.subDescription,
+            })(<TextArea rows={4} placeholder="请输入至少五个字符" />)}
+          </FormItem>
+        </Form>
+      );
+    };
+    return (
+      <>
+        <PageHeaderWrapper>
+          <div className={styles.standardList}>
+            <Card bordered={false}>
+              <Row>
+                <Col sm={8} xs={24}>
+                  <Info title="我的待办" value="8个任务" bordered />
+                </Col>
+                <Col sm={8} xs={24}>
+                  <Info title="本周任务平均处理时间" value="32分钟" bordered />
+                </Col>
+                <Col sm={8} xs={24}>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['magic numbers' are not allowed: 24](https://app.codacy.com/app/afc163/ant-design-pro/pullRequest?prid=4583897)